python/lsst/obs/base/test/dummycam.yaml

```diff
@@ -0,0 +1,133 @@
+# This file is part of obs_lsst.
+#
+# Developed for the LSST Data Management System.
+# This product includes software developed by the LSST Project
+# (http://www.lsst.org).
+# See the COPYRIGHT file at the top-level directory of this distribution
+# for details of code ownership.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the LSST License Statement and
+# the GNU General Public License along with this program.  If not,
+# see <http://www.lsstcorp.org/LegalNotices/>.
+#
+#
+---
+#
+# Simplest possible single chip camera
+
+name : DummyCam
+
+#
+# A single amplifier ("segment" to the camera team)
+#
+AMP : &AMP
+    perAmpData : True
+    # trimmed
+    dataExtent : [512, 2002]
+    readCorner : LL
+    #                         [[x0,  y0], [xsize, ysize]]
+    rawBBox                 : [[0,    0], [576,    2048]]  # total size of one amp's raw data
+
+    rawDataBBox             : [[10,   0], [512,    2002]]  # data region in raw data
+    rawSerialPrescanBBox    : [[0,    0], [10,     2002]]  # serial prescan (often == extended)
+    rawSerialOverscanBBox   : [[522,  0], [54,     2002]]  # serial overscan
+    rawParallelPrescanBBox  : [[0,    1], [0,         0]]  # pixels digitised before first parallel transfer
+    rawParallelOverscanBBox : [[10, 2002], [512,     46]]  # parallel overscan
+
+    #nRowPreParallelTransfer : 0         # number of rows digitised before first parallel transfer
+    #extended : 10                       # number of pixels in the extended register
+    #extended0 : 0                       # first pixel in the extended register
+    #h_overscan : 22                     # number of overscan pixels in x (serial)
+    #h_overscan0 : 0                     # first overscan pixel in x (serial)
+    #v_overscan : 46                     # number of overscan pixel in y (parallel)
+    #v_overscan0 : 0                     # first overscan pixel in y (parallel)
+
+    saturation : 142857                  # saturation level, DN XXX Should this be in electrons?
+
+    # Linearity correction is still under discussion, so this is a placeholder.
+    linearityType : PROPORTIONAL
+    linearityThreshold : 0
+    linearityMax : 142857                # == saturation
+    linearityCoeffs : [0, 142857]        # == [linearityThreshold, linearityMax]
+
+#
+# A single CCD
+#
```

Comment:
Do we want to formalize this three-# comment style for yaml, or is this a holdover from something else? It always felt excessively tall when I saw it in python.

Reply:
It's whatever is in the obs_lsst YAML camera. I don't like it either so I'll remove it.